test/e2e/discovery/headless_services.go

```diff
@@ -332,7 +332,7 @@ func verifyHeadlessSRVRecordsWithDig(f *framework.Framework, cluster framework.C
 func createSRVQuery(f *framework.Framework, port *corev1.ServicePort, service *corev1.Service,
 	domain string, clusterName string, withPort, withcluster bool,
 ) (cmd []string, domainName string) {
-	cmd = []string{"dig", "+short", "SRV"}
+	cmd = []string{"dig", "+short", "+timeout=20", "SRV"}
```

Comment:
Perhaps we should  make this a constant in _framework.go_ , ie `DIG_TIMEOUT = "+timeout=20"`?